task_10/src/components/FavoriteCities.js

```diff
@@ -1,16 +1,13 @@
-import {currentUserPosition, favoriteCities} from "../utils/settings";
-import {getForecastFromApi} from "../utils/api";
-import {HandlingURL} from "../utils/Url";
-import {setCoordinatesToMapStorage} from "../utils/setCoordinates";
-import {RecentlyCities} from "./RecentlyCities";
+import {favoriteCities} from "../utils/settings";
+import {geocodCityName} from "../utils/geocoding";
 
 class FavoriteCities {
     constructor() {
-        this.state = {
-            isValid: true
-        };
+        this.state = {};
```

Comment:
You can remove this at all because you not using it.

No need for creating unused variables. Also, `this.state = {}` anyway must be in a `Component` constructor.
So after calling of `super()` you will have access to `this.state`, which will be received from `Component`.